src/components/Info.js

```diff
@@ -0,0 +1,206 @@
+      mapLines.pop();
+    }
+  }
+  //open guess box and make sure county and town data are loaded
+  function guess() {
+    setGuessDepth(1);
+    setCounty(geo.address.county);
+    setTown(
+      geo.address.city ||
+        geo.address.town ||
+        geo.address.village ||
+        geo.address.hamlet
+    );
+  }
+
+  return (
+    <div id={"infoBox"}>
+      <div id={"latLongContainer"}>
+        <p id={"latDisplay"}>Lat: {displayLong}</p>
+        <p id={"longDisplay"}>Long: {displayLat}</p>
+      </div>
+      <p id={"countyTownDisplay"}>
+        {displayCounty}, {displayTown}
+      </p>
+      <p id={"scoreDisplay"}>Score: {score}</p>
+      <div id={"directionButtons"}>
+        <DirectionButton
+          direction={"North"}
+          viewLat={viewLat}
+          viewLong={viewLong}
+          setViewLat={setViewLat}
+          setViewLong={setViewLong}
+          mapObj={props.mapObj}
+          score={score}
+          setScore={setScore}
+          gameRunning={gameRunning}
+          mapLines={mapLines}
+        />
+        <DirectionButton
+          direction={"East"}
+          viewLat={viewLat}
+          viewLong={viewLong}
+          setViewLat={setViewLat}
+          setViewLong={setViewLong}
+          mapObj={props.mapObj}
+          score={score}
+          setScore={setScore}
+          gameRunning={gameRunning}
+          mapLines={mapLines}
+        />
+        <DirectionButton
+          direction={"South"}
+          viewLat={viewLat}
+          viewLong={viewLong}
+          setViewLat={setViewLat}
+          setViewLong={setViewLong}
+          mapObj={props.mapObj}
+          score={score}
+          setScore={setScore}
+          gameRunning={gameRunning}
+          mapLines={mapLines}
+        />
+        <DirectionButton
+          direction={"West"}
+          viewLat={viewLat}
+          viewLong={viewLong}
+          setViewLat={setViewLat}
+          setViewLong={setViewLong}
+          mapObj={props.mapObj}
+          score={score}
+          setScore={setScore}
+          gameRunning={gameRunning}
+          mapLines={mapLines}
+        />
+        <button id={"returnButton"}
+          onClick={() => {
+            props.mapObj.setView([long, lat]);
+            setViewLong(long);
+            setViewLat(lat);
+          }}
+          disabled={!gameRunning}
+        >
+          Return
+        </button>
+      </div>
+
+      <div id={"gameControlButtons"}>
+        <button onClick={startGame} disabled={gameRunning}>
+          Start
+        </button>
+        <button onClick={guess} disabled={!gameRunning}>
+          Guess
+        </button>
+        <button onClick={quitGame} disabled={!gameRunning}>
+          Quit
+        </button>
+      </div>
+      <GuessBox
+        depth={guessDepth}
+        setDepth={setGuessDepth}
+        county={county}
+        quit={quitGame}
+        score={score}
+        setScore={setScore}
+      />
+    </div>
+  );
+}
+
+export default Info;
```

Comment:
This file (and others) are really utilizing State (specifically the useState hook) from this weeks lectures. Plenty of state and props passing here to show a solid grasp on passing information between components (including child components).